packages/modules/desktop-viewer-react/src/services/Initializer.ts

```diff
@@ -76,7 +75,7 @@ export class DesktopInitializer {
         // register extensions before startup
         // TODO only build time ATM
         if (options?.extensions) {
-          yield addExtensions(options.extensions);
+          yield IModelApp.extensionAdmin.addExtensions(options.extensions);
```

Comment:
should we remove this yield?

Reply:
Yes my mistake I forgot the desktop initialization entirely...